File: runelite-client/src/main/java/net/runelite/client/plugins/microbot/magic/orbcharger/enums/Teleport.java
package net.runelite.client.plugins.microbot.magic.orbcharger.enums;

import lombok.Getter;
import lombok.RequiredArgsConstructor;
import net.runelite.client.plugins.microbot.util.bank.enums.BankLocation;


@Getter
@RequiredArgsConstructor
public enum Teleport {
    
    AMULET_OF_GLORY(BankLocation.EDGEVILLE, "^Amulet of glory\\(\\d\\)$"),
    RING_OF_DUELING(BankLocation.FEROX_ENCLAVE, "^Ring of dueling\\(\\d\\)$");
    
    private final BankLocation bankLocation;
    private final String regexPattern;
}

File: runelite-client/src/main/java/net/runelite/client/plugins/autoVorkath/TELEPORT.kt
package net.runelite.client.plugins.autoVorkath

import lombok.Getter
import lombok.RequiredArgsConstructor

@Getter
@RequiredArgsConstructor
enum class TELEPORT(val teleportName: String) {
    CONSTRUCT_CAPE_T("Construct. Cape (t)"),
    CONSTRUCT_CAPE("Construct. Cape"),
    HOUSE_TAB("Teleport to house");

    override fun toString(): String { return teleportName }
}
